src/components/experiment-tracking/run-dataset/run-dataset.js

```diff
@@ -99,14 +121,25 @@ const RunDataset = ({
 function buildDatasetDataMarkup(
   datasetKey,
   datasetValues,
+  datasetType,
   rowIndex,
   isSingleRun,
   pinnedRun,
   enableShowChanges,
-  selectedRunIds
+  selectedRunIds,
+  setRunDatasetToShow,
+  setShowRunVizModal
 ) {
   const updatedDatasetValues = fillEmptyMetrics(datasetValues, selectedRunIds);
   const runDataWithPin = resolveRunDataWithPin(updatedDatasetValues, pinnedRun);
+  const isPlotly = getShortType(datasetType) === 'plotly';
+  const isImage = getShortType(datasetType) === 'image';
+  const isTracking = getShortType(datasetType) === 'tracking';
```

Comment:
Nit: I'd add `dataset` to these three variable names.

```suggestion
  const isPlotlyDataset = getShortType(datasetType) === 'plotly';
  const isImageDataset = getShortType(datasetType) === 'image';
  const isTrackingDataset = getShortType(datasetType) === 'tracking';
```